### ISSUE TITLE
Display action/control/property start time in detail execution screen + Add tooltips.

### ISSUE THREAD

**Issue by abourdon** [vertigo17/Cerberus#846]
As written from #830, Step's execution time is not only related to action/control execution time but also related to the associated property computation time.

Problem is, when using automatic property computation (through `%%` and not by using the `calculateProperty` action), then it is hard to follow the Step execution time detail. 

To be more understandable, it could be great to add the property computation time from each associated action/control, for instance by adding the associated property computation time from the corresponding column (in red in the following screenshot):

![capture d ecran 2016-07-04 a 16 51 26](https://cloud.githubusercontent.com/assets/1511715/16564450/d5b60a24-4207-11e6-9bfb-74705f289c8a.png)

We could just write the total time to compute associated properties. So, if an action requires 2 properties to be computed, then just the total time will be displayed. This information could be displayed as the following:

_47.3s
(including 40s of property computation)_

If properties have already been computed, then we could just write 0s:

_47.3s
(including 0s of property computation)_

Let me know,
Aurélien

**Comment by vertigo17** [comment]
For me we don't have the information of which property were calculated during the action / control so it seems not realistic.
What we can do to help understanding is to solve the bug on that page where the start of the action / control / property calculation is displayed at 'null'.

**Comment by vertigo17** [comment]
This is what needs to be fixed :
![selection_111](https://cloud.githubusercontent.com/assets/5342048/16598481/88b93b54-42fe-11e6-9a1b-c4a71a5fb93c.jpg)
![selection_112](https://cloud.githubusercontent.com/assets/5342048/16598482/88be13a4-42fe-11e6-8092-55337c46db75.jpg)
![selection_113](https://cloud.githubusercontent.com/assets/5342048/16598483/88c17968-42fe-11e6-84a3-03ee43550c0a.jpg)
Checking the code will clarify what should be displayed. I think we do not display the ful timestamp but probably only the HH:MM:SS or even MM:SS

**Comment by vertigo17** [comment]
We also need to put the tootips on Actions/control like already exist on Properties :
![selection_123](https://cloud.githubusercontent.com/assets/5342048/16598756/acb34c7e-42ff-11e6-8bf6-bf8842b693ef.jpg)
![selection_124](https://cloud.githubusercontent.com/assets/5342048/16598757/acb8dd60-42ff-11e6-90e0-d5c33b387f48.jpg)
